_posts/en/newsletters/2021-02-10-newsletter.md

```diff
@@ -79,7 +79,13 @@ BOLTs][bolts repo].*
 
 - [HWI #433][] trezor: Enable processing of OP_RETURN outputs FIXME:Xekyo
 
-- [Bitcoin Core #19509][] Per-Peer Message Capture FIXME:adamjonas
+- [Bitcoin Core #19509][] adds per-peer message capture between nodes as well
+  as the ability to produce JSON outputs from those logs. Using the newly
+  introduced debug-only command line argument `capturemessages`, any message
+  that the node sends or receives is logged. There is a
+  [long history][bitcoin dissector] of libraries creating similar
```

Comment:
By the name of this link, I assume it's refering to Wireshark dissectors.  I'd personally refer to Wireshark as a tool (a packet inspector, to be precise) rather than a library.